apispec/core.py

```diff
@@ -16,20 +16,30 @@
     'options',
 ]
 
-OPENAPI_VERSION = SWAGGER_VERSION = '2.0'
+OPENAPI_VERSIONS = ('2.0', '3.0.0')
```

Comment:
It would be nice if users could specify just the `major` or `major.minor` version , e.g. `2`, `3`, `3.0`.

Maybe you could use `distutils.version.LooseVersion` to parse the version.